modules/userId/eids.js

```diff
@@ -42,12 +42,12 @@ const USER_IDS_CONFIG = {
 
   // liveIntentId
   'lipb': {
-    getValue: function(data) {
+    getValue: function (data) {
```

Comment:
Standard javascript style typically leaves out the space between the function keyword and the param list. We should be consistent with style that the rest of the project has.